extensions/azurePublish/src/components/azureProvisionDialog.tsx

```diff
@@ -211,6 +221,28 @@ const reviewCols: IColumn[] = [
   },
 ];
 
+const getHostname = (config) => {
+  if (config?.hostname) {
+    return config.hostname;
+  } else if (config?.name) {
+    return config?.environment ? `${config.name}-${config.environment}` : config.name;
+  }
+};
+
+const getDefaultFormData = (currentProfile, defaults) => {
+  return {
+    choice: defaults.choice ?? 'create',
```

Comment:
Nit: A more descriptive name than choice would help here.  I can find a better name in a later PR.